src/Main.java

```diff
@@ -1,5 +1,51 @@
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input the book title");
+        Book book = new Book(scanner.nextLine());
+
+        FileReader fileReader;
+        try {
+            fileReader = new FileReader(book.getFile());
+            System.out.println(book.uniqueWords(fileReader));
+            fileReader.close();
+            fileReader = new FileReader(book.getFile());
+            System.out.println(book.mostPopular(fileReader));
+            fileReader.close();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Not very  descriptive exception, probably we do not want to show an exception to the user

Reply:
ok, will fix